src/main/java/com/google/research/bleth/simulator/DummyBeacon.java

```diff
@@ -0,0 +1,38 @@
+package com.google.research.bleth.simulator;
+
+/**
+ * A Class used for testing board visualization.
+ */
+public class DummyBeacon implements Agent {
+
+    private final int id;
+    private Location realLocation;
+
+    public DummyBeacon(int id, Location realLocation) {
```

Comment:
Consider making the `Beacon` aware of its `RealBoard` owner (so it will update its own location on the real board whenever its internal location changes).

Reply:
The dummy beacon always stays at the same location, therefore I didn't think it was necessary (the actual beacons and observers are aware of their `Simulation` owner, and therefore their owner `Real Board` as well).